### ISSUE TITLE
Icon for elements

### ISSUE THREAD

**Issue by bossloper** [Nicola1971/MODxFLAT-Theme#5]
I was not sure where you got your technical.png from that is used for Elements on the Manager home page. Was it one you made, I could not find it in font awesome?

An icon will be needed for the up-coming site tree icon for, 'Manage Elements'. I thought about the 'cog' but then it would be inconsistent with the technical.png.

**Comment by Nicola1971** [comment]
3.0 uses fontawesome Wrench icon in welcome page instead technical.png .
Elements icon in tree, "was" the same used in my **tree elements plugin:**
http://fontawesome.io/icon/list-alt/

![elements](https://cloud.githubusercontent.com/assets/7342798/16914547/d586726e-4cf2-11e6-96ca-5800de7b548c.png)


But i've just replaced elements icon in tree to fit new welcome element icon
from 
http://fontawesome.io/icon/list-alt/
to
http://fontawesome.io/icon/wrench/

![wrench-icon](https://cloud.githubusercontent.com/assets/7342798/16914477/69294cfe-4cf2-11e6-8391-5130b2f0951f.png)

**Comment by bossloper** [comment]
OK, got it!
Thanks

**Comment by bossloper** [comment]
I would suggest using http://fontawesome.io/icon/files-o/ instead of http://fontawesome.io/icon/file/ so that it is different to the 'New Resouce' icon.

Also would http://fontawesome.io/icon/file-image-o/ look better than http://fontawesome.io/icon/picture-o/ for 'manage images' - a more similar shape to the other icons.

**Comment by Nicola1971** [comment]
totally agree: looks much better!
![new-buttons](https://cloud.githubusercontent.com/assets/7342798/16917581/7b8876f8-4d02-11e6-83b6-084d79acae2c.png)

Done:
https://github.com/Nicola1971/MODxFLAT-Theme/commit/7e411407ca0016f80cfba8739efd3a1584beaae5

Done:
https://github.com/Nicola1971/MODxFLAT-Theme/commit/7e411407ca0016f80cfba8739efd3a1584beaae5

**Comment by Nicola1971** [comment]
not sure about "files mangement icon": it's the same used for "duplicate resource"
![fontawicon](https://cloud.githubusercontent.com/assets/7342798/16917709/09390166-4d03-11e6-8270-713f286a7f55.jpg)

**Comment by bossloper** [comment]
Hmm. Had not noticed it was used for duplicate resource. How about http://fontawesome.io/icon/file-text-o/ for 'manage files'?

**Comment by bossloper** [comment]
or you could use http://fontawesome.io/icon/clone/ for 'duplicate resource' instead.

**Comment by Nicola1971** [comment]
clone icon done: https://github.com/Nicola1971/MODxFLAT-Theme/commit/3ccbce3e91d262b9a3cc09887b06fab7587ad952

**Comment by bossloper** [comment]
looks good.

For the right click on the menu tree: Have you made the icons 'bold' in some way on the later version?
To me it looks 'fuzzy' compared to the older version I am using and so not so 'crisp'.

**Comment by Nicola1971** [comment]
should be the opposite:

2.0 version use .png images of font awesome in context menu
3.0 version use real font awesome icon fonts

https://github.com/Nicola1971/MODxFLAT-Theme/blob/3.0/manager/media/style/MODxFLAT/style.php#L29

**Comment by bossloper** [comment]
yes, the real font awesome 'fa-lg' gives a larger icon but to me it looks softer and not so precise. I think only the 'fa-nx' classes give a nice result.

**Comment by Nicola1971** [comment]
Check the New 3.0 beta 3.5 with a lot of new features:
https://github.com/Nicola1971/MODxFLAT-Theme/issues/7

https://github.com/Nicola1971/MODxFLAT-Theme/issues/8

https://github.com/Nicola1971/MODxFLAT-Theme/issues/9
